src/workflows/airqo_etl_utils/weather_data_utils.py

```
@@ -7,8 +7,9 @@
 from .airqo_api import AirQoApi
 from .bigquery_api import BigQueryApi
 from .config import configuration
-from .constants import DataSource, Tenant
+from .constants import DataSource, DataType, Frequency, DeviceCategory
 from .data_validator import DataValidationUtils
```

Ruff (0.8.2)

`.data_validator.DataValidationUtils` imported but unused

Remove unused import: `.data_validator.DataValidationUtils`

(F401)